arch/arm/src/armv7-a/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+############################################################################
+# arch/arm/src/armv7-a/CMakeLists.txt
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+# The vector table is the "head" object, i.e., the one that must forced into
+# the link in order to draw in all of the other components
+
+set(SRCS arm_vectortab.S)
+
+# Common assembly language files
+
+list(APPEND SRCS
+  arm_cpuhead.S arm_vectoraddrexcptn.S
```

Comment:
This is still clumped - it needs to be done everywhere 1 file per line 

list(APPEND SRCS
  arm_cpuhead.S 
  arm_vectoraddrexcptn.S
...
.cp15_invalidate_dcache.S)

Reply:
The format of the file include is the same as the previous Make.defs. I don't think there is any problem with defining multiple files in one line,

Reply:
We can further optimize in subsequent commits if you insist on doing this change, since it will be a lot of work to do similar modifications

Reply:
This is a missive change. Doing another one after instead of doing it right now is not a clean transition.  You can script the change and it will not be a lot of work.

Reply:
There are more important features need to support at this stage,
kernel/protect build, export build, custom board build, port process flow, armcc/clang support, xtensa/risc-v support ....
I do not want to waste time tweaking the format of file include since which style is already exists in Make.defs,
please let me know if there is any standard that mentioned the CMake must be define the file line by line.

Reply:
https://github.com/cheshirekow/cmake_format

Reply:
Thanks for the share, seems [cmake_format](https://github.com/cheshirekow/cmake_format) to be a good style format tool, let me unified the format!

Reply:
https://github.com/apache/incubator-nuttx/pull/6718/commits/7655a0387bfff04da808a9d5079ef802a9c019d3